### PR TITLE
Kubevirt: Fix vm tempalte breadcrumbs

### PR DESCRIPTION
**Pull request by yaacov**
Follow-up to #1843, fix breadcrumbs label and link.

Ref:
https://github.com/openshift/console/pull/1843#issuecomment-508179319

Before (link to vm template list is broken):
OKD(2)

After:
OKD(1)

### CODE REVIEW

**Comment by yaacov** [comment]
@mareklibra @vojtechszocs @suomiy @rawagner @matthewcarleton please review.

**Comment by matthewcarleton** [comment]
thanks @yaacov!
just a nit: Can we ensure the 'vm-name-example details' is properly capitalized: 'Vm-name-example Details' This could be handled in the css rather than the html to be more bullet proof.

**Comment by yaacov** [comment]
> Can we ensure the 'vm-name-example details' is properly capitalized: 'Vm-name-example Details' 

:+1: Thanks, changed `details` to `Details`

> This could be handled in the css rather than the html to be more bullet proof.

For future references (not for this PR) the css for breadcrumb is here:
https://github.com/openshift/console/blob/master/frontend/public/components/_nav-title.scss#L32
Adding `text-transform: capitalize` may be a nice touch :-)

**Comment by yaacov** [comment]
@matthewcarleton ^^

File: frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template.tsx [hunk 1]
```diff
@@ -156,4 +156,4 @@ type VirtualMachineTemplatesPageProps = {
   match: match<{ ns?: string }>;
 };
 
-export { VirtualMachineTemplatesPage };
+export { labelPlural, VirtualMachineTemplatesPage };
```
Comment (suomiy) [inline frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template.tsx]: this should probably go to some other file like `./constants.ts`

Comment (yaacov) [reply frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template.tsx]: :+1: nice

**Pull request merged**